.golangci.yml

```diff
@@ -1,18 +1,21 @@
 # Documentation: https://golangci-lint.run/usage/configuration/
 linters-settings:
   errcheck:
-    exclude: .errcheck.txt
+    exclude-functions:
```

Comment:
a little surprised to see the list hasn't changed in four years 🙃 

https://github.com/tektoncd/pipeline/pull/867/files#diff-abc3252939fdc262df264426e7345f18778ea70ba9ea174dc97435bb6f729a9e

Reply:
That makes me happy! We haven't enabled additional exclusions!